frontend/sw360-portlet/src/main/java/com/siemens/sw360/portal/portlets/components/ComponentPortlet.java

```diff
@@ -527,9 +526,7 @@ private void prepareReleaseDetailView(RenderRequest request, RenderResponse resp
 
             if (!isNullOrEmpty(releaseId)) {
                 release = client.getReleaseById(releaseId, user);
-                final Set<String> licenseNames = new HashSet<>(SW360Utils.getLicenseNamesFromIds(release.getMainLicenseIds(), user.getDepartment()));
                 final Set<License> licenses = new HashSet<>(SW360Utils.getLicenses(release.getMainLicenseIds(), user.getDepartment()));
```

Comment:
Now that you've removed `release.setMainLicenses(licenses);`, `licenses` is never used anymore